mappings/net/minecraft/screenhandler/slot/Slot.mapping

```diff
@@ -1,11 +1,10 @@
-CLASS net/minecraft/class_1735 net/minecraft/container/Slot
+CLASS net/minecraft/class_1735 net/minecraft/screenhandler/slot/Slot
 	FIELD field_7871 inventory Lnet/minecraft/class_1263;
 	FIELD field_7872 yPosition I
 	FIELD field_7873 xPosition I
 	FIELD field_7874 id I
 	FIELD field_7875 invSlot I
 	METHOD <init> (Lnet/minecraft/class_1263;III)V
-		ARG 1 inventory
```

Comment:
Why is the argument deleted?

Reply:
it is redundant as it will be inferred from the class name